seaborn/categorical.py

```diff
@@ -1498,24 +1498,52 @@ def estimate_statistic(self, estimator, ci, n_boot):
             self.value_label = "{}({})".format(estimator.__name__,
                                                self.value_label)
 
-    def draw_confints(self, ax, at_group, confint, colors, **kws):
-
-        kws.setdefault("lw", mpl.rcParams["lines.linewidth"] * 1.8)
+    def draw_confints(self,
+                      ax, at_group,
+                      confint,
+                      colors,
+                      conf_lw=None,
```

Comment:
Please change here and in the external interface to `errwidth` for consistency with `errcolor`.